helper.js

```diff
@@ -0,0 +1,9 @@
+const multiplyByTwo = (arr) => {
+  const multiplied = [];
+
+  for (const el of arr) {
+    multiplied.push(el * 2);
+  }
+
+  return multiplied;
+}
```

Comment:
Consider switching to the map function

[MDN Docs for .map](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/map)